code/modules/vending/vending.dm

```diff
@@ -2116,7 +2121,9 @@
 	desc = "One stop shop for all your custodial needs."
 	icon_state = "janitor"
 	icon_panel = "standard-panel"
-	icon_deny = "null"
+	icon_off = "standard-off"
+	icon_broken = "standard-broken"
+	icon_fallen = "standard-fallen"
```

Comment:
?

Reply:
The JaniTech vendor had a custom sprite for the regular version, but the off/broken/fallen states still used the old sprite, which had a different-colored metal frame. I added new sprites for the off/broken/fallen JaniTech, so I think that's what this particular change is aimed at?

Reply:
but why is it `standard-`

Reply:
yeah my bad, I thought the JaniTech didnt have a differently coloured frame so it could just use the standard sprite. Fixed it now so it should use the proper one